app/controllers/home_controller.rb

```diff
@@ -10,6 +10,17 @@ def index
   def me
   end
 
+  def edit_me
```

Comment:
what about making a `ProfilesController` or `UserProfilesController` and having this be a `show` action instead? That would be more Railsy (although that can be considered a compliment or a dis, depending on who you are :hamburger: )